src/site/stages/build/process-cms-exports/schemas/output/node-event_listing.js

```diff
@@ -0,0 +1,62 @@
+const { usePartialSchema } = require('../../transformers/helpers');
+const eventSchema = require('./node-event');
+
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-event_listing'] },
+    entityType: { type: 'string', enum: ['node'] },
+    entityBundle: { type: 'string', enum: ['event_listing'] },
+    entityUrl: { $ref: 'EntityUrl' },
+    title: { type: 'string' },
+    created: { type: 'number' },
+    changed: { type: 'number' },
+    entityMetatags: { $ref: 'MetaTags' },
+    entityPublished: { type: 'boolean' },
+    fieldAdministration: { $ref: 'output/taxonomy_term-administration' },
+    fieldDescription: { type: 'string' },
+    fieldIntroText: { type: 'string' },
```

Comment:
Based on this error:
<img width="919" alt="Screen Shot 2020-08-06 at 8 42 12 AM" src="https://user-images.githubusercontent.com/55560129/89536970-a631cf00-d7c6-11ea-8952-83e22b02bfd5.png">


```suggestion
    fieldIntroText: { type: ['string', 'null'] },
```